.github/workflows/create-release.yml

```diff
@@ -352,7 +352,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.20.x
```

Comment:
these changes get overwritten by whatever we have in https://github.com/paketo-buildpacks/github-config/tree/main/stack, but we should update this to be `stable` there anyway